packages/markup-include/src/_stories/MarkupInclude.stories.tsx

```diff
@@ -1,18 +1,316 @@
+            columnName: 'Location',
+            conditions: []
+          },
+          {
+            name: 'url',
+            tag: '{{url}}',
+            columnName: 'URL',
+            conditions: []
+          }
+        ],
+        showHeader: true,
+        srcUrl: '#example',
+        title: 'Current Rate by Location',
+        useInlineHTML: true
+      },
+      data: [
+        {
+          STATE: 'Alabama',
+          Rate: '130',
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Alaska',
+          Rate: '80',
+          Location: 'Home',
+          URL: 'https://www.cdc.gov/'
+        }
+      ],
+      legend: {},
+      newViz: true,
+      theme: 'theme-blue',
+      type: 'markup-include',
+      runtime: null,
+      visual: {
+        border: false,
+        accent: false,
+        background: false,
+        hideBackgroundColor: false,
+        borderColorTheme: false
+      }
+    },
+    isEditor: false,
+    showEditorPanel: true
+  }
+}
+
+export const With_conditions: Story = {
+  args: {
+    config: {
+      contentEditor: {
+        inlineHTML: '<div>{{state}} does have a rate 130 compared to the over all rate of {{overall-rate}}</div>',
+        markupVariables: [
+          {
+            name: 'state',
+            tag: '{{state}}',
+            columnName: 'STATE',
+            conditions: [{ columnName: 'Rate', isOrIsNotEqualTo: 'is', value: '130' }]
+          },
+          {
+            name: 'overall-rate',
+            tag: '{{overall-rate}}',
+            columnName: 'Rate',
+            conditions: [{ columnName: 'STATE', isOrIsNotEqualTo: 'is', value: 'Overall' }]
+          }
+        ],
+        showHeader: true,
+        srcUrl: '#example',
+        title: '',
+        useInlineHTML: true
+      },
+      data: [
+        {
+          STATE: 'Overall',
+          Rate: '80',
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        },
+        {
+          STATE: 'Alabama',
+          Rate: '130',
+          Location: 'Vehicle',
+          URL: 'https://www.cdc.gov/'
+        }
+      ],
+      legend: {},
+      newViz: true,
+      theme: 'theme-amber',
+      type: 'markup-include',
+      runtime: null,
+      visual: {
+        border: false,
+        accent: true,
+        background: true,
+        hideBackgroundColor: false,
+        borderColorTheme: false
+      }
+    },
+    isEditor: false,
+    showEditorPanel: true
+  }
+}
+
+export const Button_and_text: Story = {
+  args: {
+    config: {
+      contentEditor: {
+        inlineHTML: '<div>{{state}} does have a rate 130 compared to the over all rate of {{overall-rate}}</div>',
+        markupVariables: [],
+        showHeader: false,
+        srcUrl: 'https://www.cdc.gov/wcms/4.0/cdc-wp/data-presentation/examples/Markup-Include-Button-and-Text.html',
+        title: '',
+        useInlineHTML: true
+      },
+      data: [

```

Comment:
make data one variable or json file and import it so we don't end up with multiple duplicates of multiline data blocks